primitives/src/signatures/mod.rs

```diff
@@ -114,6 +139,43 @@ mod tests {
         assert!(S::verify(&parameters, &pk, message, &sig).is_ok());
     }
 
+    pub(crate) fn agg_sign_and_verify<S: AggregateableSignatureSchemes>(
+        messages: &[&[S::MessageUnit]],
+        bad_message: &[S::MessageUnit],
+    ) {
+        let rng = &mut test_rng();
+        let parameters = S::param_gen(Some(rng)).unwrap();
+        let mut sks = vec![];
```

Comment:
What is the purpose of `sks`?

Reply:
Good point, we don't need to store `sks`. Fixed in 90bd4e62fe6da77347006baeb1eb141a62e35817